spec/javascripts/unit/save-location.spec.js

```diff
@@ -1,184 +1,181 @@
 describe('Saving bank holiday locations', function () {
-  var $locations
-  var html = '<section class="js-save-nation" data-module="save-bank-holiday-nation" data-nation="England_and_Wales" data-nation-matches=\'["England","Wales"]\' data-save-description="Do you want to save # as your location for bank holidays?" data-save-button-aria-label="Save # as your location for bank holidays" data-save-button-text="Save" data-undo-description="You\'ve saved # as your location for bank holidays" data-undo-button-aria-label="Remove # as your saved location for bank holidays" data-undo-button-text="Undo">' +
-    '<p class="js-nation-description" aria-live="polite">Do you want to save # as your location for bank holidays?</p>' +
-    '<button class="js-nation-button" type="submit" aria-label="Save # as your location for bank holidays">Save</button>' +
-    '<p class="js-nation-link">' +
-      '<a href="#" class="govuk-link">We\'ll add a cookie to your device</a>' +
-    '</p>' +
-  '</section>'
+  var element
 
   beforeEach(function () {
     window.GOVUK.setCookie('cookies_policy', '{"essential":true,"settings":true,"usage":true,"campaigns":true}')
+    element = document.createElement('section')
+    element.classList.add('js-save-nation')
+    element.setAttribute('data-nation', 'England_and_Wales')
+    element.setAttribute('data-nation-matches', '["England","Wales"]')
+    element.setAttribute('data-save-description', 'Do you want to save # as your location for bank holidays?')
+    element.setAttribute('data-save-button-aria-label', 'Save # as your location for bank holidays')
+    element.setAttribute('data-save-button-text', 'Save')
+    element.setAttribute('data-undo-description', 'You\'ve saved # as your location for bank holidays')
+    element.setAttribute('data-undo-button-aria-label', 'Remove # as your saved location for bank holidays')
+    element.setAttribute('data-undo-button-text', 'Undo')
+    element.innerHTML =
+      '<p class="js-nation-description" aria-live="polite">Do you want to save # as your location for bank holidays?</p>' +
+      '<button class="js-nation-button" type="submit" aria-label="Save # as your location for bank holidays">Save</button>' +
+      '<p class="js-nation-link">' +
+        '<a href="#" class="govuk-link">We\'ll add a cookie to your device</a>' +
+      '</p>'
   })
 
   afterEach(function () {
     window.GOVUK.deleteCookie('cookies_policy')
     window.GOVUK.deleteCookie('user_nation')
-    $locations.remove()
     window.location.hash = ''
   })
 
   describe('initial state', function () {
-    beforeEach(function () {
-      $locations = $(html)
-      $('body').append($locations)
-    })
-
     it('shows the bank holiday option if cookies are allowed', function () {
-      new GOVUK.Modules.SaveBankHolidayNation($locations[0]).init()
-      expect($locations[0].style.display).toEqual('block')
+      var bank = new GOVUK.Modules.SaveBankHolidayNation(element)
+      bank.init()
```

Comment:
Bit of a nitpick, but on the other lines these two lines are combined.

Reply:
Ah yes thanks, Fixed.